exporter/awsemfexporter/grouped_metric.go

```diff
@@ -83,6 +97,138 @@ func addToGroupedMetric(pmd *pdata.Metric, groupedMetrics map[interface{}]*Group
 			}
 		}
 	}
+
+	return nil
+}
+
+type kubernetesObj struct{
+	ContainerName string `json:"container_name"`
+	Docker internalDockerObj `json:"docker"`
+	Host string `json:"host"`
+	Labels internalLabelsObj `json:"labels"`
+	NamespaceName string `json:"namespace_name"`
+	PodId string `json:"pod_id"`
+	PodName string `json:"pod_name"`
+	PodOwners internalPodOwnersObj `json:"pod_owners"`
+	ServiceName string `json:"service_name"`
+}
+
+type internalDockerObj struct{
+	ContainerId string `json:"container_id"`
+}
+
+type internalLabelsObj struct{
+	App string `json:"app"`
+	PodTemplateHash string `json:"pod-template-hash"`
```

Comment:
ditto

Reply:
use "_" for jason attribute name since you had it everywhere else

Reply:
@mxiamxia unfortunately I can't do this as I need to match the format which cloud watch expects